src/onboarding/onboarding-core-config.ts

```diff
@@ -131,6 +131,16 @@ class OnboardingCoreConfig extends LitElement {
       this._save(ev);
       return;
     }
+
+    // Set suggested country
+    if (navigator.language) {
+      let suggested = navigator.language.split("-")[1].toUpperCase();
+      if (!COUNTRIES.includes(suggested)) {
+        suggested = "NL";
```

Comment:
People don't read the message and will click through. We should not make false assumptions or give the use the impression  that no action is required.

Reply:
if a user clicked through, the default location is set to NL. It is better to have a value picked, than no value. Currently it looks like you can choose not to pick a country, which is not the case.

Reply:
The onboarding will force or block the user until they set it correctly. After this PR, if they don't notice, they'll be in the country of their browser language or in the Netherlands if that doesn't mention anything. 

Some settings are based on what country is being chosen, therefore I think it's safer to leave it empty until the user notices they can't move forward and fills it in correctly.

Reply:
The country picker is only used for the units. Picking the Dutch units will pick Celsius, 24h, etc, which are also the most common. They can always be changed later too.

Reply:
Removed it 👍